### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by marema31**
Creating a docker image

### CODE REVIEW

**Comment by nqb** [comment]
Hello @marema31,

Thanks for your PR. It will be interesting to be able to start a Docker image with a specific version of Venom.

**Comment by yesnault** [comment]
Hi @marema31 ,

Before making a release, we do a lot of testing to get the most stable version possible. 

We don't want to promote the master branch as a stable branch for 'production' use. The master branch can contain some breaking changes, it's not a good idea to run venom as is, without knowing which changes can break your existing tests.

Here, a dockerfile with the latest release:

```dockerfile
FROM debian:buster-slim

RUN apt-get update && \
    apt-get install -y curl unixodbc && \
    rm -rf /var/lib/apt/lists/*

RUN LAST_RELEASE=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep tag_name | head -n 1 | cut -d '"' -f 4) && \
    todl=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep ${LAST_RELEASE} | grep browser_download_url | grep -E 'venom.linux-amd64' | cut -d '"' -f 4) && \
    curl -s $todl -L -o /opt/venom && \
    chmod +x /opt/venom

ENTRYPOINT ["/opt/venom" ]

CMD [ "run", "--output-dir", "/outputs"]
```

And we will provide official image on docker hub for the next release.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Default volume for tests output and logs
+VOLUME /outputs
+
+#Default volume for tests suite
+VOLUME /testsuite
+WORKDIR /testsuite
+
+ENTRYPOINT ["/opt/venom" ]
+
+CMD [ "run", "--output-dir", "/outputs"]
```
Comment (yesnault) [inline Dockerfile:28]: FROM debian:buster-slim

RUN apt-get update && \
    apt-get install -y curl unixodbc && \
    rm -rf /var/lib/apt/lists/*

RUN LAST_RELEASE=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep tag_name | head -n 1 | cut -d '"' -f 4) && \
    todl=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep ${LAST_RELEASE} | grep browser_download_url | grep -E 'venom.linux-amd64' | cut -d '"' -f 4) && \
    curl -s $todl -L -o /opt/venom && \
    chmod +x /opt/venom

ENTRYPOINT ["/opt/venom" ]

CMD [ "run", "--output-dir", "/outputs"]

**Review by yesnault — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (yesnault) [inline Dockerfile:28]:
```suggestion
FROM debian:buster-slim

RUN apt-get update && \
    apt-get install -y curl unixodbc && \
    rm -rf /var/lib/apt/lists/*

RUN LAST_RELEASE=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep tag_name | head -n 1 | cut -d '"' -f 4) && \
    todl=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep ${LAST_RELEASE} | grep browser_download_url | grep -E 'venom.linux-amd64' | cut -d '"' -f 4) && \
    curl -s $todl -L -o /opt/venom && \
    chmod +x /opt/venom

ENTRYPOINT ["/opt/venom" ]

CMD [ "run", "--output-dir", "/outputs"]
```

**Review by yesnault — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -35,6 +35,12 @@ Flags:
       --var-from-file strings  --var-from-file filename.yaml --var-from-file filename2.yaml : hcl|json|yaml, must contains map[string]string'
 ```
 
+## Docker image
+
+A docker image is available.
+```bash
+  docker run -it --rm -v $(pwd):/tests run /tests/testsuite.yaml
```
Comment (marema31) [inline README.md:42]: It should also mount the /outputs volume to obtains the tests results, no ?

Comment (yesnault) [reply README.md:42]: yes

File: README.md [hunk 3]
```diff
@@ -207,6 +214,7 @@ Builtin variables:
 
 ## Export test results as jUnit, json, yaml or tap reports
 
+
```
Comment (yesnault) [inline README.md:217]:
```suggestion
```

**Review by yesnault — APPROVED** [review]

**Pull request merged**